service-manual/domain-names/https.md

```diff
@@ -62,7 +62,7 @@ The verification methods that are commonly in use (in order of preference) are:
 2. Create a special DNS record with a code supplied by the SSL vendor
 3. Sending an email to the owner of the service.gov.uk domain
 
-The least preferred method of sending an email to the domain owner relies on the GDS Infrastructure Team
+The least preferred method of sending an email to the domain owner relies on the GOV.UK Infrastructure Team
```

Comment:
OK. Would you care to take a crack at rewording the last sentence in this paragraph, which feels a little tortuous?